salt/returners/mysql.py

```diff
@@ -261,7 +265,8 @@ def _get_serv(ret=None, commit=False):
                                    passwd=_options.get('pass'),
                                    db=_options.get('db'),
                                    port=_options.get('port'),
-                                   ssl=ssl_options)
+                                   ssl=ssl_options,
+                                   unix_socket=_options.get('unix_socket'))
```

Comment:
From glimpsing at the lib I can't really tell how MySQLdb is dealing with the kwargs here. What's happening if you pass `host`/`port` and `unix_socket` - which one is getting priority?

Reply:
from `this` explanation i get that `host` has always priority over `unix_socket`, only not when it is `localhost`. then `unix_port` snaps either with the default or the passed argument.